contracts/modules/UpgradableModuleFactory.sol

```diff
@@ -0,0 +1,88 @@
+pragma solidity ^0.5.0;
+
+import "./ModuleFactory.sol";
+import "../proxy/OwnedUpgradeabilityProxy.sol";
+
+/**
+ * @title Factory for deploying upgradable modules
+ */
+contract UpgradableModuleFactory is ModuleFactory {
+
+    event LogicContractSet(string _version, address _logicContract, bytes _logicData);
+
+    struct LogicContract {
+        string version;
+        address logicContract;
+        bytes logicData;
```

Comment:
Maybe we should ask issuers to input `logicData`? If we store it like this, we won't be able to pass custom parameters.

Reply:
My thinking here was:
  - logicData is for the upgrade process (determined by the factory owner) - e.g. if some data needs to be fixed as a part of the upgrade.
  - the issuer (i.e. ST) wouldn't be able to re-initialise the module as part of the upgrade process (which may break some internal logic on the module)